### ISSUE TITLE
Another victim here. Very disappointed

### ISSUE THREAD

**Issue by greggluhring** [bitpay/copay#8646]
I sent ~ $250 in BCH to a BCH wallet address in CoPay and the money has disappeared into thin air. I now kick myself that I didn't do a small amount of research into the CoPay reputation before diving in and blindly throwing my coins into the bitpay ether.

Every case I've read here ends the same way unsuccessfully. I've yet to find someone who walked away with their cash. I hope someone here can help me do that.

I'm curious why the Wallet Service URL (https://bws.bitpay.com/bws/api) is not available. Is that an issue?
 If the app can't get blockchain information how can it function?

Please help me.

**Comment by greggluhring** [comment]
Yes, I do have a questio/problem with copay. I sent $250 worth of BCH to the BCH wallet address in CoPay and it has not arrived. In my experience BCH is very quick in comparison to BTC eg; 10 minutes compared to 40 min? It's been 20 hours since I sent the funds and there is no sign of them.

I sent 0.1958808 BCH to the address 16KM4r69Lew3arqJGC2kjosxcDFvJv5NMT given to me by CoPay (about $248) and there is no sign of it in CoPay but the address at blockchair shows $276 going in and coming out.

https://blockchair.com/bitcoin-cash/transaction/df69c7c7c7ca7570c2fa7274ad1ec575512644ba3cb2946fcf592ac436d7297a

I see it in there 
https://blockchair.com/bitcoin-cash/address/qqa9ph0r8w6jmhj8gvzckdhf535hh6dljggvw564cd
but why doesn't copay take it out?

How can I get it back? Thank you for you help.

Gregg

**Comment by BGalliotti** [comment]
Hello @greggluhring

Your case is, you need to recover some BCH funds from a Copay wallet right?

In that case, you can follow #8586. If you did not have a BCH wallet you will have to duplicate the BTC wallet and then recover the funds. All of that is explained in #8586 

Please try that solution and let us know what happens.
Kind regards.

**Comment by Richie202a** [comment]
@greggluhring 
if you couldnt recover your lost fund you can contact me I can get it back : anywalletrecovery@gmail.com

**Comment by gfxguru** [comment]
So glad I stopped using bitpay. So many people losing btc/money from this platform. People I suggest you get your funds and don't look back.
